test/query_test.rb

```diff
@@ -32,9 +39,21 @@ def test_find
     assert_equal [], User.find(name: "Noah").to_a
   end
 
+  def test_find!
+    user = User.create(name: "Bryan")
+    assert_equal user, User.find!(user.id)
+    assert_raises(Minidoc::DocumentNotFoundError) { User.find!(BSON::ObjectId.new) }
```

Comment:
I'm wondering if this assertion should pass or fail:

```ruby
assert_raises(Minidoc::DocumentNotFoundError) { User.find!(_id: BSON::ObjectId.new) }
```

I think it won't, but maybe should for consistency. WDYT?